NNPTPZ1/ComplexNumber.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+
+namespace Mathematics
+{
+    public class ComplexNumber
+    {
+        public double RealNumber { get; set; }
+        public double ImaginaryUnit { get; set; }
+
+        public readonly static ComplexNumber Zero = new ComplexNumber()
+        {
+            RealNumber = 0,
+            ImaginaryUnit = 0
+        };
+
+        public ComplexNumber Add(ComplexNumber c)
+        {
+            return new ComplexNumber()
+            {
+                RealNumber = RealNumber + c.RealNumber,
+                ImaginaryUnit = ImaginaryUnit + c.ImaginaryUnit
+            };
+        }
+        public ComplexNumber Subtract(ComplexNumber c)
+        {
+            return new ComplexNumber()
+            {
+                RealNumber = RealNumber - c.RealNumber,
+                ImaginaryUnit = ImaginaryUnit - c.ImaginaryUnit
+            };
+        }
+
+        public ComplexNumber Multiply(ComplexNumber c)
+        {
+            return new ComplexNumber()
+            {
+                RealNumber = RealNumber * c.RealNumber - ImaginaryUnit * c.ImaginaryUnit,
+                ImaginaryUnit = RealNumber * c.ImaginaryUnit + ImaginaryUnit * c.RealNumber
+            };
+        }
+
+        internal ComplexNumber Divide(ComplexNumber c)
+        {
+            ComplexNumber dividend = Multiply(new ComplexNumber() { RealNumber = c.RealNumber, ImaginaryUnit = -c.ImaginaryUnit });
+            double denominator = c.RealNumber * c.RealNumber + c.ImaginaryUnit * c.ImaginaryUnit;
+
+            return new ComplexNumber()
+            {
+                RealNumber = dividend.RealNumber / denominator,
+                ImaginaryUnit = dividend.ImaginaryUnit / denominator
+            };
+        }
+
+        public double GetAbsolute()
```

Comment:
wrong name, absolute value is technical term in complex numbers and should be preserved without any abbreviations or shortenings